pkg/get/tools.go

```diff
@@ -836,12 +836,42 @@ https://github.com/{{.Owner}}/{{.Repo}}/releases/download/{{.Version}}/{{.Name}}
 			Repo:  "opa",
 			Name:  "opa",
 			BinaryTemplate: `{{ if HasPrefix .OS "ming" -}}
-		{{.Name}}_windows_amd64.exe
-		{{- else if eq .OS "darwin" -}}
-		{{.Name}}_darwin_amd64
-		{{- else -}}
-		{{.Name}}_linux_amd64
-		{{- end -}}`,
+			{{.Name}}_windows_amd64.exe
+			{{- else if eq .OS "darwin" -}}
+			{{.Name}}_darwin_amd64
+			{{- else -}}
+			{{.Name}}_linux_amd64
+			{{- end -}}`,
+		})
+
+	tools = append(tools,
```

Comment:
Just curious  and I'm not asking you to change this, but do we not need, or can we not specify a version?

Reply:
Since this is a custom example, could you add a unit test too?

Reply:
We can fix the version, but when I looked into minio doc they have strategy to place their latest release at the given url. Sure, will add unit test case for it.

Reply:
Sounds good to me.

Reply:
Done

Reply:
@alexellis

Reply:
Thanks